dmorg.tex

```diff
@@ -0,0 +1,97 @@
+
+As shown in \figref{fig:dmorg}, the organization now features lead institutions, each with responsibility for major element of the DM System (Level 2 Work Breakdown Structure elements).
+For example, during Final Design, the Process Control and Archive Site Manager and Team at NCSA will be conducting prototyping activities in computing, data communications, and data storage to select and verify the ability of System technologies to support the LSST requirements.
+They will also be involved in creating a supporting infrastructure for the DM Systems.
+During Construction, before the LSST first light time frame, these resources will be focused on implementation of the selected technologies.
+In order to ensure that team functions as one integrated project, the institutions coordinate support by other lead institution team members directly through this organisational structure, as well as via a number of cross-organisational bodies (described later in this document).
+
+\begin{figure}[htbp]
+\begin{center}
+ \includegraphics[width=\textwidth]{images/DmOrg}
+\caption{DM organisation with Scientists in Green. \label{fig:dmorg}}
+\end{center}
+\end{figure}
+
+
+\subsection {Document Management} \label{sect:docman}
+
+DM documents will follow the System Engineering Guidelines of LSST. PDF versions of released documents shall be put in Docushare in accordance with the Project's Document Management Plan \citedsp{LPM-51}.
+
+The Document Tree for DM is shown in \figref{fig:doctree}, it is not exhaustive but gives a high level orientation for the main documents in DM and how they relate to each other.
+
+\begin{figure}
+\begin{center}
+ \includegraphics[width=0.8\textwidth]{images/DocTree}
+\caption{Outline of the documentation tree for DM software relating the high level documents to each other. \label{fig:doctree}}
+\end{center}
+\end{figure}
+
+{\bf Need a DOC tree for End User Documentaton - Jonathan ?}
+
+{\bf Need a DOC tree for Hardware/Services  - Margaret }
+
+\subsubsection{Draft Documents}
+
+Draft DM documents will be kept in GitHub. A single repository per document will be maintained with the head revision containing the {\em released } version which should match the version on docushare. Each repository will be included as a {\em submodule} of a single git repository located at \url{https://github.com/lsst-dm/dm-docs}.
+
+Use of Google Docs or confluence is tolerated but final delivered documents must conform to the standard LSST format, and hence either produced with LaTeX, using the lsst-texmf package\footnote{\url{https://lsst-texmf.lsst.io}}, or Word, using the appropriate LSST template \citedsp{Document-9224, Document-11920}. The precurosr document should then be erased with a pointer to the baseline document, stored in GitHub.
```

Comment:
I'm in favour of converting all the reStructuredText LDMs to LaTeX; I don't know who will do this conversion.